design/FY2019/NFP-HAMT-Enhancements-Pt1.md

```diff
@@ -0,0 +1,307 @@
+       \note refers to the list of layers specified for the construction applied to the above surface
+       \note If a source occurs on a layer surface, the layer should be split into two layers (thin where source occurs) and the source added to the thin layer
+  A2,  \field Source Type
+       \required-field
+       \type choice
+       \key UserDefined
+       \key StackAndOverPressure
+       \key AirflowNetwork
+       \minimum 0
+       \maximum 3
+       \note Type of Moisture Source Calculation
+       \note UserDefined: User defined input of air flow through component
+       \note StackAndOverPressure: Calculation of dynamic air flow through component due to stack effect and ventilation overpressure (according to Kuenzel, Zirkelbach and Schafazcek 2012)
+       \note AirflowNetwork: Calculation of dynamic air flow through component with AirflowNetwork model
+  N6,  \field Air Flow Rate
+       \units m/s
+       \type real
+       \minimum -0.001
+       \maximum 0.001
+       \note Air flux density in m3 per m2 and second.
+       \note Only required for Source Type UserDefined.
+  N7,  \field Stack height
+       \units m
+       \type real
+       \minimum 0
+       \maximum 100
+       \note Height of the connected airspace in the building envelope element
+       \note Only required for Source Type StackAndOverPressure.
+  N8,  \field Component Air Permeance
+       \units m/s
+       \type real
+       \minimum 0
+       \maximum 1
+       \note Moisture specific air permeance of the component in m3 per m2, hour and Pascal.
+       \note Only required for Source Type StackAndOverPressure.
+  N9,  \field Mechanical Ventilation Overpressure
+       \units Pa
+       \type real
+       \minimum 0
+       \maximum 100
+       \note Constant mechanical ventilation overpressure in Pascal.
+       \note Only required for Source Type StackAndOverPressure.
+  A3,  \field Outside Layer
+       \required-field
+       \type object-list
+       \object-list MaterialName
+  A4,  \field Layer 2
+       \type object-list
+       \object-list MaterialName
+  A5,  \field Layer 3
+       \type object-list
+       \object-list MaterialName
+  A6,  \field Layer 4
+       \type object-list
+       \object-list MaterialName
+  A7,  \field Layer 5
+       \type object-list
+       \object-list MaterialName
+  A8,  \field Layer 6
+       \type object-list
+       \object-list MaterialName
+  A9,  \field Layer 7
+       \type object-list
+       \object-list MaterialName
+  A10, \field Layer 8
+       \type object-list
+       \object-list MaterialName
+  A11, \field Layer 9
+       \type object-list
+       \object-list MaterialName
+  A12; \field Layer 10
+       \type object-list
+       \object-list MaterialName
+```
+
+### Modify Existing Object, Add Two New Objects ###
+
+This option modifies the existing sources object to move the source information (both thermal and moisture) out into separate objects.
+
+```
+Construction:InternalHeatSource,
```

Comment:
I lean toward this format, but suggest ConstructionProperty:InternalHeatSource, have it reference the construction name and material layer number and then the existing Construction:InternalSource transitions to plain old Construction plus the new ConstructionProperty:InternalHeatSource. 

OR if you don't want to touch Construction:InternalSource, how about two new object just for HAMT that are parallel to the existing HAMT objects:
MaterialProperty:HeatAndMoistureTransfer:HeatSource
MaterialProperty:HeatAndMoistureTransfer:MoistureSource

Reply:
OK, I'll make some edits with your naming and go with the "separate objects" plan.